lib/index.js

```diff
@@ -61,18 +67,19 @@ function TradeOfferManager(options) {
 	}
 
 	if (this._steam) {
-		this._steam.on('tradeOffers', (count) => {
+		this._steam.on('tradeOffers', () => {
```

Comment:
I understand that some inspections dislike when you have unused function parameters, but I prefer to fill in all parameters if it's an event where we don't control the emitting. Someday, we may wish to use `count` but it may not be apparently obvious that it's available without referencing documentation.